core/src/main/java/net/opentsdb/pools/BlockingQueueArrayObjectPool.java

```diff
@@ -103,8 +103,10 @@ public PooledObject claim(final long time, final ChronoUnit unit) {
   @Override
   public PooledObject claim(final int length) {
     if (length > ((ArrayObjectPoolAllocator) config.allocator()).pooledLength()) {
-      tsdb.getStatsCollector().incrementCounter("objectpool.claim.largeArray", 
+      tsdb.getStatsCollector().incrementCounter("objectpool.claim.lengthTooBig.count", 
           "pool", config.id());
+      tsdb.getStatsCollector().setGauge("objectpool.claim.lengthTooBig.size", 
```

Comment:
This is the metric to track array size.  👍

Reply:
The `Gauge` metric will give the count as well. We don't need another `Counter` metric